pkg/virt-controller/services/template_test.go

```diff
@@ -4281,6 +4281,59 @@ var _ = Describe("Template", func() {
 			)
 
 		})
+		Context("with configmap in VMI annotations for generic sidecar", func() {
+			var vmi *v1.VirtualMachineInstance
+
+			BeforeEach(func() {
+				vmi = api.NewMinimalVMI("generic-sidecar-test")
+				vmi.Annotations = map[string]string{
+					hooks.HookSidecarListAnnotationName: `[{"image": "test:test", "configMap": {"name": "test-cm", "key": "script.sh"}}]`,
+				}
+			})
+			When("ConfigMap exists on the cluster", func() {
+				BeforeEach(func() {
+					k8sClient := k8sfake.NewSimpleClientset()
+					//virtClient.EXPECT().CoreV1().Return(cm).MaxTimes(1)
```

Comment:
Can remove this, no?

Reply:
Thanks for the catch. During my `git rebase` adventure that you recommended, I specifically searched for this one. Maybe I searched for it too soon. 😉

Reply:
an unsolicited tip: you can git blame to see where it was introduced, then rebase up to that point and remove it.

Reply:
@maiqueb thanks! Didn't strike me earlier. I'll keep that in mind. :)